### ISSUE TITLE
Unlock any pop up blocking attempts

### ISSUE THREAD

**Issue by kaiodcosta** [mozilla-mobile/fenix#17225]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have exactly one feature request described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Feature Requests are better when they’re open-ended instead of demanding a specific solution e.g: “I want an easier way to do X” instead of “add Y”)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### What is the user problem or growth opportunity you want to see solved? 

The functionality of unlock any Pop Up Blocking Attempts. That is, the function would block any attempt to open pop-up requests, accepting any pop-ups from all sites or only those selected.

### How do you know that this problem exists today? Why is this important? 

This problem was described in a question on mozilla support about the functionality to unblock any attempts to block pop-ups. This functionality is important because without it, certain website functions will be limited and will have to use other browsers or means to access this website with the pop-up. Technically, this functionality already exists in other versions of the application such as for IOS and computer for example.

### Who will benefit from it? 

Users who use certain sites like work who directly require this function.
